makefiles/kconfig.mk

```diff
@@ -18,7 +18,7 @@ CFLAGS += -include '$(KCONFIG_GENERATED_AUTOCONF_HEADER_C)'
 
 # Header for the generated header file
 define KCONFIG_AUTOHEADER_HEADER
-/* RIOT Configuration File */
+/// RIOT Configuration File
```

Comment:
Not sure if this would work, but to keep doxygen style maybe try this:

```suggestion
ifeq ($(OS),Cygwin)
//* RIOT Configuration File *//
else
/* RIOT Configuration File */
endif
```

Reply:
In my case `$(OS)` is `MINGW32_NT-6.2`, is there a way to detect this in a more generic way?

Reply:
Hmm I'm not sure, for now I guess `ifneq (,$(filter Cygwin MINGW%,$(OS)))` could work, but I'm not sure if uname is present everyware. BTW how are you building in windows?

Reply:
> but I'm not sure if uname is present everyware

I think it's safe to use `uname`.

> BTW how are you building in windows?

After struggling a bit I managed to set up a VM with mingw, msys and the ARM toolchain.

Reply:
In lack of a better source possible uname outputs: https://es.wikipedia.org/wiki/Uname, many more cases to cover. The question here is which Kernel's do we support? I would say `ifeq ($(OS),CYGWIN% MINGW% MSYS%)` should be enough.

Reply:
So the alternative could be something like this (`ifeq` is not allowed inside the `define` block):

```
$(if $(filter MINGW% CYGWIN% MSYS%,$(OS)),/,)/* RIOT Configuration File */
```

Reply:
Yes that looks good to me, you can squash after fixing, but lets wait a bit to see if we can get confirmation from @kenrabold before merging if thats OK with you!

Reply:
Updated